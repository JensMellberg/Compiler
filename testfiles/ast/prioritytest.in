int a()
	{
x = 5 + 5 * 5;
x = 5 * 5 + 5;
x = 5 + 5 * 5 + 5;
x = (5 + 5) * 5;
}
