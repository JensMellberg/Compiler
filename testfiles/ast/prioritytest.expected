Program
  List
    FuncDecl
      IdDecl ID="a"
      List
      Block
        List
          Assign
            IdUse ID="x"
            Add
              Numeral NUMERAL="5"
              Mul
                Numeral NUMERAL="5"
                Numeral NUMERAL="5"
          Assign
            IdUse ID="x"
            Add
              Mul
                Numeral NUMERAL="5"
                Numeral NUMERAL="5"
              Numeral NUMERAL="5"
          Assign
            IdUse ID="x"
            Add
              Add
                Numeral NUMERAL="5"
                Mul
                  Numeral NUMERAL="5"
                  Numeral NUMERAL="5"
              Numeral NUMERAL="5"
          Assign
            IdUse ID="x"
            Mul
              Add
                Numeral NUMERAL="5"
                Numeral NUMERAL="5"
              Numeral NUMERAL="5"
